SPEC.md

```diff
@@ -130,6 +134,11 @@ The network configuration is described in JSON form. The configuration can be st
   - `routes` (list): List of subnets (in CIDR notation) that the CNI plugin should ensure are reachable by routing them through the network. Each entry is a dictionary containing:
     - `dst` (string): subnet in CIDR notation
     - `gw` (string): IP address of the gateway to use. If not specified, the default gateway for the subnet is assumed (as determined by the IPAM plugin).
+- `dns`: Dictionary with DNS specific values:
+  - `nameservers` (list of strings): list of a priority-ordered list of DNS nameservers that this network is aware of. Each entry in the list is a string containing either an IPv4 or an IPv6 address.
+  - `domain` (string): the local domain used for short hostname lookups.
```

Comment:
Thanks, I will add a note about how  most resolvers handle this.

Some background info:
The idea is to have a convenient way to pass a domain to the runtime which might want to set a new domain name in case it created a new UTS namespace. How runtimes handle multiple networks that pass along a domain is beyond scope though, as well as how it handles the case of *domain* and *search* given. Personally I'd prepend *domain* to *search* if it's not already included and use this as the new *search*.